scss/foundation.scss

```diff
@@ -77,6 +78,7 @@
     @include foundation-flex-grid;
   }
   @include foundation-typography;
+  @include foundation-dropcaps;
```

Comment:
Please move it to `@mixin foundation-typography` in `scss/typography/_typography.scss`